pkg/artifactory/resource/configuration/resource_artifactory_ldap_setting.go

```diff
@@ -115,14 +115,14 @@ func ResourceArtifactoryLdapSetting() *schema.Resource {
 		"search_filter": {
 			Type:             schema.TypeString,
 			Optional:         true,
-			ValidateDiagFunc: validation.ToDiagFunc(validation.All(validation.StringIsNotEmpty, utils.ValidateLdapFilter)),
+			ValidateDiagFunc: validation.ToDiagFunc(validation.All(validation.StringIsNotEmpty, ValidateLdapFilter)),
```

Comment:
Move this to validators

Reply:
We really should consider contributing - I know the new Validator types haven't been ported from the old yet

Reply:
Also, since this is the only kind of validator we use, let's just make it named and not re-wrap